web/assets/scss/main.scss

```diff
@@ -53,21 +53,22 @@ body:not(.js-enabled) .govuk-back-link {
 }
 
 .app-button-link {
```

Comment:
for some reason I just forgot we had this style, I've added `display: contents` so it wraps in text like a link would